src/WaveTrack.cpp

```diff
@@ -981,9 +1035,73 @@ void WaveTrack::ClearAndPaste(double t0, // Start of time to clear
       // Restore cut/split lines
       if (preserve) {
 
-         // Restore the split lines, transforming the position appropriately
-         for (const auto split: splits) {
-            SplitAt(warper->Warp(split));
+         auto attachLeft = [](WaveClip* target, WaveClip* src) 
```

Comment:
Making the second clip const would be nice, and also in the other lambda following